soft4pes/sim/simulation.py

```diff
@@ -35,11 +72,24 @@ def __init__(self, sys, conv, ctr):
             Converter model.
         ctr : controller object
             Control system.
+        Ts : float
+            Simulation sampling interval [s].
         """
         self.sys = sys
         self.conv = conv
         self.ctr = ctr
+        self.Ts = Ts
         self.t_stop = 0
+        self.matrices = self.sys.get_discrete_state_space(
+            self.conv.v_dc, self.Ts)
+
+        # Check if self.ctr.Ts/Ts is an integer. Use tolerance to prevent
+        # floating point errors
+        Ts_rat = self.ctr.Ts / self.Ts
```

Comment:
It would be better to define the control sampling interval as Tc to avoid confusion.

Reply:
I changed this to simulation sampling to Ts_sim, as Ts is already used in control systems and discretizations. Ts is also used in (at least our) publications for control systems, so I would rather keep that as it is, so that the articles and the code match